### ISSUE TITLE
[$10][MSFT-29] [Challenge Feed] "Topcoder 2019" contrast between foreground and background colors won't meet WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2751]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges and Login using valid credentials
2. In the footer; check "Topcoder 2019"

**Expected Result**
Expected contrast ratio of 4.5:1

- Passes AA, if you change background color to #3F3F3F. New contrast ratio: 4.54
- Passes AA, if you change text color to #6A6A6C. New contrast ratio: 4.53

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

**Actual Result**
Element has insufficient color contrast of 1.94 (foreground color: #aaaaab, background color: #ebebeb, font size: 9.8pt, font weight: normal).

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F13

**Comment by ThomasKranitsas** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096163 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by r0hit-gupta** [comment]
Please assign me

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096163 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by r0hit-gupta** [comment]
@veshu how do I self assign myself from now onwards? I cannot see the option.

**Comment by veshu** [comment]
@r0hit-gupta Can you please check with this link https://x.topcoder.com/api/v1/github/teams/registration/e07056ab-46c5-407d-9323-1f20de914a6e-1563396610310

**Comment by r0hit-gupta** [comment]
@veshu it says `{"version":"v3","result":{"success":false,"status":403,"content":{"message":"Failed to authenticate token."}}}`

**Comment by veshu** [comment]
@r0hit-gupta how about this https://x.topcoder.com/api/v1/github/teams/registration/462d863d-9f1a-4179-a215-29f71a551e4f-1563464121672 ?

**Comment by r0hit-gupta** [comment]
@veshu same result. I have Github 2FA already enabled.

**Comment by veshu** [comment]
@Dara-K Can we use this color b8b8b9 for this one? 
![image](https://user-images.githubusercontent.com/2202045/61687482-37f8a680-ad42-11e9-83a4-eed5056fdc60.png)


Please suggest the color for this. Thanks!

**Comment by Dara-K** [comment]
Please use #AAAAAB. The above color is not in the gray palette we use.

**Comment by veshu** [comment]
@r0hit-gupta did you changed on react-ui-kit? if so please reference the PR here to verification by architect too.

**Comment by r0hit-gupta** [comment]
@veshu I haven't made any changes yet. Waiting for the reply from @Dara-K. Will refer the new PR once done.

**Comment by veshu** [comment]
Where does this color variable defined then $tc-gray-30? per codemintor this should be defined on react ui kit. @r0hit-gupta

**Comment by r0hit-gupta** [comment]
@veshu it is already defined in react UI kit. Please review. The change should pass the AA requirement now.

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2751 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097523 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097523 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
Test URL : https://test-community-app.topcoder-dev.com/challenges

**Comment by drasticdpk** [comment]
As suggested by @lakshmiathreya, Verified the color code  provided by designer on production env.
Operating System: Windows
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097523<br/><br/>```This is an automated message for crazyk via Topcoder X```
